cumulus/client/cirrus-executor/src/processor.rs

```diff
@@ -98,6 +99,7 @@ where
 		primary_hash: PHash,
 		bundles: Vec<OpaqueBundle>,
 		shuffling_seed: Randomness,
+		maybe_new_runtime: Option<Cow<'static, [u8]>>,
```

Comment:
You can replace this with `Vec<u8>`, the only reason it is like that in runtime API is to avoid allocation in runtime internally. Outside it'll always be vector.

Reply:
We can keep it as maybe someday the static wasm slice only needs to be converted to a Vector when it's really necessary even outside the runtime :D